app/mailers/verification_code_mailer.rb

```diff
@@ -6,11 +6,11 @@ def with_code
     @locale = params[:locale]
     @verification_code = params[:verification_code]
     attachments.inline['logo.png'] = service.email_logo
-    if @service_type == :statefile
-      @subject = I18n.t("messages.verification_code_subject_with_service_name", service_name: @service_name, locale: @locale)
-    else
-      @subject = I18n.t("messages.default_subject_with_service_name", service_name: @service_name, locale: @locale)
-    end
+    @subject = if @service_type == :statefile
+                 I18n.t('messages.verification_code_subject_with_service_name', service_name: @service_name, locale: @locale)
```

Comment:
Where is `@locale` coming from? I think part of this story is to collect the locale from the language setting when the user is on on questions/contact-preference and keep that as the stored locale/language.

Reply:
Rails parses the URL and adds it to the params hash in `params[:locale]` line 6 of this file.

Reply:
@mpidcock 👆🏼

Reply:
oh nice, it's already there 👍